FindMe/View Controllers/FindMeViewController.m

```diff
@@ -52,6 +53,7 @@ - (IBAction)onFindMe:(id)sender
         if ([self.locationManager authorisationStatus] == AllowedAlways) {
             [self designTrackingButtonWithState:@"selected"];
             [self.locationManager beginTracking];
```

Comment:
If I were you I'd create a Trace object that holds its own creation date and pass that to the location manager and database manager. It's always a good idea to figure out what the bundle of data looks like and pass that around instead of random pieces. You are almost doing this right now so this shouldn't be a large refactor.